testplan/testing/multitest/base.py

```diff
@@ -804,7 +804,7 @@ def _run_suite_related(self, testsuite, method):
         return method_report
 
     def _run_case_related(self, method, testcase, case_result):
-        interface.check_signature(method, ["name", "self", "env", "result"])
```

Comment:
i think the pre_testcase funcs we are using in our tests/examples do has a signature of name, self, env, result.

Reply:
Only the decorator `@pre_testcase` and `@post_testcase` is used in tests, I do not see examples having this yet.

Reply:
I see, i think your change is correct and no wonder it didn't break any test - because def pre_testcase way of defining pre_testcase is never used/tested.